nvi-test/src/main/java/no/sikt/nva/nvi/test/TestUtils.java

```diff
@@ -106,7 +92,8 @@ public static Candidate randomCandidate() {
     }
 
     public static ApprovalStatus randomApprovalStatus() {
-        return new ApprovalStatus(randomUri(), randomElement(Status.values()), randomUsername(), Instant.EPOCH);
+        return new ApprovalStatus(randomUri(), randomElement(Status.values()), BigDecimal.ONE, randomUsername(),
```

Comment:
Should use bigger values for BigDecimal - on both side of the decimal . That way we can test that the conversion back/forth dynamodb works as expected in CandidateTest.
BigDecimal<->String<->AttributeValue(number)

Reply:
Added `randomBigDecimal()` for testing with a random values and a scale of 10. The specs for nvi points requires a scale of 4, but might be changed to 7 in the future.